test/e2e/common/privileged.go

```diff
@@ -75,7 +75,7 @@ func (c *PrivilegedPodTestConfig) run(containerName string, expectSuccess bool)
 		Expect(err).NotTo(HaveOccurred(),
 			fmt.Sprintf("could not remove dummy1 link: %v", err))
 	} else {
-		Expect(err).To(HaveOccurred(), msg)
+		framework.ExpectError(err, msg)
```

Comment:
We can remove import gomega :-)
```
ERROR(all) test/e2e/common/privileged.go:23:2: "k8s.io/kubernetes/vendor/github.com/onsi/gomega" imported but not used exit status 1 !!! Type Check has failed. This may cause cross platform build failures. !!! Please see https://git.k8s.io/kubernetes/test/typecheck for more information.
```

Reply:
Thank you for your comment.
I removed it.